dell-csi-helm-installer/csi-offline-bundle.sh

```diff
@@ -218,11 +224,25 @@ set_mode() {
 CREATE="false"
 PREPARE="false"
 REGISTRY=""
+DRIVER="csi-powerstore"
 
 # some directories
 SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 REPODIR="$( dirname "${SCRIPTDIR}" )"
-HELMDIR="${REPODIR}/helm"
+if [ ! -d "$REPODIR/helm-charts" ]; then
+  
+  if  [ ! -d "$SCRIPTDIR/helm-charts" ]; then
+    git clone --quiet -c advice.detachedHead=false -b csi-powerstore-2.7.0 https://github.com/dell/helm-charts
+  fi
+  mv helm-charts $REPODIR
+else 
+  if [  -d "$SCRIPTDIR/helm-charts" ]; then
+    rm -rf $SCRIPTDIR/helm-charts
+  fi
+fi
+
+# HELMDIR="${REPODIR}/helm"
```

Comment:
@sakshi-garg1 can you please remove this comment if its not required?

Reply:
@suryagupta4  sure